acceptance/tests/sameness/sameness_test.go

```diff
@@ -0,0 +1,445 @@
+					Namespace:   staticServerNamespace,
+				}
+
+				// Sameness Defaults need to be applied first so that the sameness group exists.
+				applyResources(t, cfg, "../fixtures/bases/mesh-gateway", members[k].context.KubectlOptions(t))
+				applyResources(t, cfg, "../fixtures/bases/sameness/default-ns", members[k].context.KubectlOptions(t))
+				applyResources(t, cfg, "../fixtures/bases/sameness/override-ns", members[k].serverOpts)
+
+				// Only assign a client if the cluster is running a Consul server.
+				if v.hasServer {
+					members[k].client, _ = members[k].helmCluster.SetupConsulClient(t, c.ACLsEnabled)
+				}
+			}
+
+			// TODO: Add further setup for peering, right now the rest of this test will only cover Partitions
+			// Create static server deployments.
+			logger.Log(t, "creating static-server and static-client deployments")
+			k8s.DeployKustomize(t, members[keyPrimaryServer].serverOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory,
+				"../fixtures/cases/sameness/static-server/default")
+			k8s.DeployKustomize(t, members[keyPartition].serverOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory,
+				"../fixtures/cases/sameness/static-server/partition")
+
+			// Create static client deployments.
+			k8s.DeployKustomize(t, members[keyPrimaryServer].clientOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory,
+				"../fixtures/cases/sameness/static-client/default")
+			k8s.DeployKustomize(t, members[keyPartition].clientOpts, cfg.NoCleanupOnFailure, cfg.NoCleanup, cfg.DebugDirectory,
+				"../fixtures/cases/sameness/static-client/partition")
+
+			// Verify that both static-server and static-client have been injected and now have 2 containers in server cluster.
+			// Also get the server IP
+			for _, labelSelector := range []string{"app=static-server", "app=static-client"} {
+				podList, err := members[keyPrimaryServer].context.KubernetesClient(t).CoreV1().Pods(metav1.NamespaceAll).List(context.Background(),
+					metav1.ListOptions{LabelSelector: labelSelector})
+				require.NoError(t, err)
+				require.Len(t, podList.Items, 1)
+				require.Len(t, podList.Items[0].Spec.Containers, 2)
+				if labelSelector == "app=static-server" {
+					ip := &podList.Items[0].Status.PodIP
+					require.NotNil(t, ip)
+					logger.Logf(t, "partition-static-server-ip: %s", *ip)
+					members[keyPrimaryServer].staticServerIP = ip
+				}
+
+				podList, err = members[keyPartition].context.KubernetesClient(t).CoreV1().Pods(metav1.NamespaceAll).List(context.Background(),
+					metav1.ListOptions{LabelSelector: labelSelector})
+				require.NoError(t, err)
+				require.Len(t, podList.Items, 1)
+				require.Len(t, podList.Items[0].Spec.Containers, 2)
+				if labelSelector == "app=static-server" {
+					ip := &podList.Items[0].Status.PodIP
+					require.NotNil(t, ip)
+					logger.Logf(t, "default-static-server-ip: %s", *ip)
+					members[keyPartition].staticServerIP = ip
+				}
```

Comment:
1. Are the IP logs here swapped w.r.t. default vs. partition?
2. (Minor) ~ Would it make sense to break this code out into a helper?

Reply:
yes you're right about the log swap. I'm not 100% sure what I'm going to do with this when I add peering as well, so that's why I didn't add a helper yet. I'll consider it when I add in peering!